### DISCUSSION TITLE
Collapse/Expand feature

### DISCUSSION THREAD

**Discussion by ademkikaj** [microsoft/fluentui#17015] [Q&A] [answered]
I am trying to implement something like the Boostrap's collapse/expand. The closest one is `Nav`, but it is very limited since I want to do a combination with `StackItem`. Any help on that?

**Comment by JustSlone** [comment, marked as answer]
This sounds like an Accordion component. You'd probably have to roll your own using a chevron and a stack or div it's fairly simple to do in limited cases. 

We are working on one for Fluent UI vNext - https://github.com/microsoft/fluentui/blob/master/packages/react-accordion/Spec.md

**Reply by JustSlone** [reply]
Also tagging @brandonthomas who might have thoughts on a good way to implement a collapse/expand

**Reply by brandonthomas** [reply]
Yah the main thing we used for this in M365 Admin was accordion as well. We implemented it using the W3C ARIA example (https://w3c.github.io/aria-practices/examples/accordion/accordion.html). We didn't use groups though and just had each accordion as a tab stop which has worked well for users so far.

**Reply by ademkikaj** [reply]
I am using FluentUI through React components only. Seems like now I have to build my own React component based on @brandonthomas answer? @JustSlone Can you please provide more info regarding Fluet UI vNext? Right now on the official [page](https://developer.microsoft.com/en-us/fluentui#/controls/web), I don't see any Accordion Control.
